Source/ParameterEncoding.swift

```diff
@@ -241,8 +241,11 @@ public struct URLEncoding: ParameterEncoding {
             let value = parameters[key]!
             components += queryComponents(fromKey: key, value: value)
         }
-
+        #if swift(>=4.0)
+        return components.map { "\($0.0)=\($0.1)" }.joined(separator: "&")
+        #else
         return components.map { "\($0)=\($1)" }.joined(separator: "&")
+        #endif
```

Comment:
To keep formatting consistent across Alamofire, how about?

```swift
    }

#if swift(>=4.0)
    return components.map { "\($0.0)=\($0.1)" }.joined(separator: "&")
#else
    return components.map { "\($0)=\($1)" }.joined(separator: "&")
#endif
```

Where the pound defines are demented one tab.